modules/geo-layers/src/tile-layer/tileset-2d.js

```diff
@@ -263,14 +263,16 @@ export default class Tileset2D {
   _getTile({x, y, z}, create) {
     const tileId = `${x},${y},${z}`;
     let tile = this._cache.get(tileId);
+    const {layerId} = this.opts;
 
     if (!tile && create) {
       tile = new Tile2DHeader({
         x,
         y,
         z,
         onTileLoad: this.onTileLoad,
-        onTileError: this.onTileError
+        onTileError: this.onTileError,
+        layerId
       });
       Object.assign(tile, this.getTileMetadata(tile));
       tile.loadData(this._getTileData);
```

Comment:
If we can introduce the `Tileset2D` concept here, we would call something like `this.state.tileset.getTiles(frame)` and that would issue the requests...